bindings/c/test/mako/mako.cpp

```diff
@@ -467,7 +464,7 @@ void dumpThreadSamples(Arguments const& args,
 				continue;
 			}
 			auto fclose_guard = ExitGuard([fp]() { fclose(fp); });
-			sample_bins[op].forEachBlock([fp](auto ptr, auto count) { fwrite(ptr, sizeof(*ptr) * count, 1, fp); });
+			stats.writeToFile(filename, op);
```

Comment:
Re-use fp above instead of filename or remove fp code. I would prefer the latter.